lib/mime.ex

```diff
@@ -32,8 +32,15 @@ defmodule MIME do
     end
   end)
 
+  old = Application.get_env(:plug, :mimes, %{})
   app = Application.get_env(:mime, :types, %{})
 
+  unless Map.keys(old) == [] do
```

Comment:
Deprecation is done inside Plug: https://github.com/elixir-lang/plug/commit/03e5efb5b957ecc250455ecaf6bc615187e21234#diff-eefdb392f58102aed281d3c5f08ad377R4.